source/agora/test/Base.d

```diff
@@ -1778,33 +1780,12 @@ public class TestValidatorNode : Validator, TestAPI
         super(config);
     }
 
-    /// Create an enrollment data used as information for an validator
-    public override Enrollment createEnrollmentData ()
+    /// ditto
+    public Enrollment toggleEnroll ()
```

Comment:
Why is it returning the enrollment ? The point of this was to have something like:
```suggestion
    public void setRecurringEnrollment (bool doIt)
```

And then toggle it on/off from test. This PR makes things better, because at least we're not duplicating code anymore, but it doesn't yet go all the way.

Reply:
We need to check all the nodes to have the enrollment which is returned from the function like as follows.
```
Enrollment new_enroll = nodes[0].setRecurringEnrollment(true);
Thread.sleep(3.seconds);  // enrollValidator() can take a while..
nodes.each!(node =>
        retryFor(node.getEnrollment(new_enroll.utxo_key) == Enrollment.init, 1.seconds));
```
I think the name of the function does not match with the operation of that too.